config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -133,7 +133,7 @@ periodics:
       - --test-cmd-args=--testoverrides=./testing/experiments/enable_prometheus_api_responsiveness.yaml
       - --test-cmd-args=--testoverrides=./testing/experiments/enable_restart_count_check.yaml
       - --test-cmd-args=--testoverrides=./testing/experiments/use_simple_latency_query.yaml
-      - --test-cmd-args=--testoverrides=./testing/overrides/kubemark/throughput_override.yaml
+      - --test-cmd-args=--testoverrides=./testing/overrides/kubemark_100_nodes.yaml
```

Comment:
Only this one is the right one - thanks!

Reply:
Actually no - this one is also not exactly what we have - see: https://github.com/kubernetes/perf-tests/pull/1300/files#r433645388

Reply:
Fixed to kubemark_load_throughput.yaml

Reply:
Thanks!